ca/server.go

```diff
@@ -373,7 +383,16 @@ func (s *Server) Run(ctx context.Context) error {
 			case state.EventUpdateCluster:
 				s.updateCluster(ctx, v.Cluster)
 			}
-
+		case <-ticker.C:
```

Comment:
I don't quite understand this. The external CA code has no reconciliation loop of its own. It just gets called to sign an individual certificate. I really don't think it's a good idea for the external CA code to know anything about which certificates are pending, and to make decisions about retries. This should all be in one place to keep things simple... otherwise we'd basically have two implementations of the CA server that behave differently. Also, I think this functionality should apply for the internal CA case as well, to handle the case where the signer is unavailable but becomes available later on.